travisci/all-housekeeping/executability.t

```diff
@@ -35,6 +35,10 @@ foreach my $f (@all_files) {
     # Exceptions
     $should_be_executable = '' if $f =~ /\/docs\/conf.py$/;
     $should_be_executable = '' if $f =~ /production.*reg_conf.*\.pl$/;
+    $should_be_executable = '' if $f =~ /docs\/conf\.py$/;
```

Comment:
Lines 36 and 38 are fairly similar. Do we need the distinction between them? I feel the latter one you have added should be enough to cover both scenarios.
```suggestion
    $should_be_executable = '' if $f =~ /docs\/conf\.py$/;
    $should_be_executable = '' if $f =~ /production.*reg_conf.*\.pl$/;
```

Reply:
Ah, I didn't see it !

Reply:
I'll fix the source commit